interface/patient_file/history/edit_billnote.php

```diff
@@ -28,59 +28,61 @@
 ?>
 <html>
 <head>
-<?php Header::setupHeader(); ?>
-
-<style>
-</style>
-
+    <?php Header::setupHeader(); ?>
 </head>
 
 <body>
-<?php
-if ($_POST['form_submit'] || $_POST['form_cancel']) {
-    if (!CsrfUtils::verifyCsrfToken($_POST["csrf_token_form"])) {
-        CsrfUtils::csrfNotVerified();
-    }
+    <?php
+    if ($_POST['form_submit'] || $_POST['form_cancel']) {
+        if (!CsrfUtils::verifyCsrfToken($_POST["csrf_token_form"])) {
+            CsrfUtils::csrfNotVerified();
+        }
 
-    $fenote = trim($_POST['form_note']);
-    if ($_POST['form_submit']) {
-        sqlStatement("UPDATE form_encounter " .
-        "SET billing_note = ? WHERE id = ?", array($fenote,$feid));
-    } else {
-        $tmp = sqlQuery("SELECT billing_note FROM form_encounter " .
-        " WHERE id = ?", array($feid));
-        $fenote = $tmp['billing_note'];
-    }
+        $fenote = trim($_POST['form_note']);
+        if ($_POST['form_submit']) {
+            sqlStatement("UPDATE form_encounter " .
+            "SET billing_note = ? WHERE id = ?", array($fenote,$feid));
+        } else {
+            $tmp = sqlQuery("SELECT billing_note FROM form_encounter " .
+            " WHERE id = ?", array($feid));
+            $fenote = $tmp['billing_note'];
+        }
 
-  // escape and format note for viewing
-    $fenote = $fenote;
-    $fenote = str_replace("\r\n", "<br />", $fenote);
-    $fenote = str_replace("\n", "<br />", $fenote);
-    if (! $fenote) {
-        $fenote = '[' . xl('Add') . ']';
-    }
+        // escape and format note for viewing
+        $fenote = $fenote;
+        $fenote = str_replace("\r\n", "<br />", $fenote);
+        $fenote = str_replace("\n", "<br />", $fenote);
 
-    echo "<script>\n";
-    echo " parent.closeNote(" . js_escape($feid) . ", " . js_escape($fenote) . ")\n";
-    echo "</script></body></html>\n";
-    exit();
-}
+        echo "<script>\n";
+        echo "dlgclose();";
+        echo "</script></body></html>\n";
 
-$tmp = sqlQuery("SELECT billing_note FROM form_encounter " .
-  " WHERE id = ?", array($feid));
-$fenote = $tmp['billing_note'];
-?>
+        exit();
+    }
 
-<form method='post' action='edit_billnote.php?feid=<?php echo attr_url($feid); ?>' onsubmit='return top.restoreSession()'>
-<input type="hidden" name="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken()); ?>" />
+    $tmp = sqlQuery("SELECT billing_note FROM form_encounter " .
+    " WHERE id = ?", array($feid));
+    $fenote = $tmp['billing_note'];
+    ?>
 
-<center>
-<textarea name='form_note' style='width:100%'><?php echo text($fenote); ?></textarea>
-<p>
-<input type='submit' name='form_submit' value='<?php echo xla('Save'); ?>' />
-&nbsp;&nbsp;
-<input type='submit' name='form_cancel' value='<?php echo xla('Cancel'); ?>' />
-</center>
-</form>
+    <div class="container">
+        <div class="row">
+            <div class="col-12">
```

Comment:
We don't need a row and col-12 here since the form is already naturally at 100% width.